src/src/editor/D2Viewer.kt

```diff
@@ -45,7 +45,9 @@ private class ProjectLevelCoroutineScopeHolder(val coroutineScope: CoroutineScop
 internal const val D2_EDITOR_NAME = "D2FileEditor"
 
 @Serializable
-internal data class D2FileEditorState(@JvmField var theme: D2Theme?, @JvmField val layout: D2Layout?) : FileEditorState {
+internal data class D2FileEditorState(@JvmField var theme: D2Theme?,
+                                      @JvmField val layout: D2Layout?,
+                                      @JvmField val sketch: Boolean?) : FileEditorState {
```

Comment:
We could specify a default value here with `@Default(false)` and make the field not nullable

Reply:
Couldn't find a suitable @Default annotation, set the default with "= false"

Reply:
I'm not familiar with kotlinx.serialization enough. Sorry for a misleading suggestion :)

The default value looks like the right solution according to the docs.